odoo-shippable/scripts/build-image.sh

```diff
@@ -81,6 +81,7 @@ add_custom_aptsource "${VIM_PPA_REPO}" "${VIM_PPA_KEY}"
 add_custom_aptsource "${TMUX_PPA_REPO}" "${TMUX_PPA_KEY}"
 
 # Release the apt monster!
+echo "resolvconf resolvconf/linkify-resolvconf boolean false" | debconf-set-selections
```

Comment:
This change is related with the follow problem 

```bash
Setting up pgdg-keyring (2017.3) ...
Setting up poppler-utils (0.24.5-2ubuntu4.8) ...
Setting up postgresql-client-common (188.pgdg14.04+1) ...
Setting up postgresql-client-9.6 (9.6.6-1.pgdg14.04+1) ...
Setting up postgresql-common (188.pgdg14.04+1) ...
invoke-rc.d: policy-rc.d denied execution of start.
Setting up postgresql-9.6 (9.6.6-1.pgdg14.04+1) ...
invoke-rc.d: policy-rc.d denied execution of start.
Setting up postgresql-contrib-9.6 (9.6.6-1.pgdg14.04+1) ...
Setting up tmux (2.0-1~ppa1~t) ...
Setting up ruby1.9.1 (1.9.3.484-2ubuntu1.5) ...
Setting up perl-modules (5.18.2-2ubuntu1.3) ...
Setting up libruby1.9.1 (1.9.3.484-2ubuntu1.5) ...
Setting up perl (5.18.2-2ubuntu1.3) ...
Setting up git (1:2.15.0-1~ppa0~ubuntu14.04.1) ...
Processing triggers for libc-bin (2.19-0ubuntu6.13) ...
Processing triggers for initramfs-tools (0.103ubuntu4.9) ...
Processing triggers for ca-certificates (20170717~14.04.1) ...
Updating certificates in /etc/ssl/certs... 17 added, 42 removed; done.
Running hooks in /etc/ca-certificates/update.d....done.
Errors were encountered while processing:
 resolvconf
E: Sub-process /usr/bin/dpkg returned an error code (1)
```
The `apt-get` package `libffi-dev` is part of the solution

https://github.com/moby/moby/issues/1297#issuecomment-115458690